Makefile

```diff
@@ -51,13 +51,24 @@ format: $(GO)
 	$(GO) fmt ./pkg/... ./cmd/...
 	$(GO) vet ./pkg/... ./cmd/...
 
-test: $(GO)
+test-build-host-local-plugin:
+	if [ ! -f $(BIN_DIR)/host-local -a `uname` = 'Linux' ]; then\
+		cd $(BIN_DIR);\
+		git clone https://github.com/containernetworking/plugins;\
+		cd plugins; git checkout v0.9.1; cd ..;\
+		./plugins/build_linux.sh;\
+		cp ./plugins/bin/host-local .;\
+		rm -rf plugins;\
```

Comment:
```suggestion
		cd $(BIN_DIR) && \
		git clone https://github.com/containernetworking/plugins && \
		pushd plugins && git checkout v0.9.1 && popd && \
		./plugins/build_linux.sh && \
		cp ./plugins/bin/host-local . && \
		rm -rf plugins; \
```

Reply:
Fixed other than ```pushd/popd``` because of the error ```/bin/sh: 4: pushd: not found```.